src/components/Stepper.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/no-unstable-nested-components */
+// We need to disable this rule because the MUI Stepper API requires us to pass a component
+// to the `slots.stepIcon` prop, and defining a simple function inline is the most
+// straightforward way to achieve this. The performance impact is minimal in this case.
+import { Stepper as MuiStepper, Step, StepLabel, StepConnector, useTheme } from '@mui/material';
+import CloudUploadIcon from '@mui/icons-material/CloudUpload';
+import ViewAgendaIcon from '@mui/icons-material/ViewAgenda';
+import FactCheckIcon from '@mui/icons-material/FactCheck';
+import CloudDownloadIcon from '@mui/icons-material/CloudDownload';
+import { View } from '../utils/types';
+
+const steps = [
+  { label: 'Upload', view: View.Upload, icon: CloudUploadIcon },
+  { label: 'Column Annotation', view: View.ColumnAnnotation, icon: ViewAgendaIcon },
+  { label: 'Value Annotation', view: View.ValueAnnotation, icon: FactCheckIcon },
+  { label: 'Download', view: View.Download, icon: CloudDownloadIcon },
+];
```

Comment:
this seems somewhat overlapping with the stuff in `types.ts` and in the `view.ts` store - isn't that information we could use across the app and thus would make sense to store centrally like that?